pdftk: add pt_BR translation (#3383)
tldr-pages/tldr: pages.pt_BR/common/pdftk.md

# pdftk

> Conjunto de utilitários para manipular ficheiros PDF.

> > Mas informação em: <https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit>.

Reviewer: ```suggestion
> Mais informações: <https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit>.
```

> - Extrair conjuntos de páginas de um ficheiro PDF (páginas 1 a 3, 5 e 6 a 10) e guardá-las num novo ficheiro:

Reviewer: Acho que a palavra "ficheiro" não é muito utilizada por quem fala Português. Pelo menos é a minha percepção, tendo o português como primeira língua. Podemos trocar?

Reviewer: Se possivel, fazer isso para todas as ocorrências no arquivo. Obrigado.

Reviewer 2: Sim, @schneiderl, eu ia comentar isso. "Ficheiro" é o termo que usamos em Portugal, mas como isto é a tradução pt_BR, faz mais sentido usar "arquivo" (quer nas descrições quer nos tokens dos comandos de exemplo).

Reviewer: Perfeito. 
Só agora fui perceber que você é de Portugal 👍

Reviewer 2: Ahahahah, vou levar isso como um elogio! 😃

`pdftk {{ficheiro.pdf}} cat {{1-3 5 6-10}} output {{novo_ficheiro.pdf}}`

> - Juntar (concatenar) uma lista de ficheiros PDF, guardando o resultado num novo ficheiro:

Reviewer: ```suggestion
- Concatenar uma lista de ficheiros PDF, guardando o resultado num novo ficheiro:
```

`pdftk {{ficheiro1.pdf ficheiro2.pdf ficheiroN.pdf ...}} cat output {{novo_ficheiro.pdf}}`

- Partir cada página de um ficheiro PDF num ficheiro separado, com um padrão para o nome dos novos ficheiros:

`pdftk {{ficheiro.pdf}} burst output {{página_%d.pdf}}`

> - Rodar todas as páginas de um ficheiro PDF 180°:

Reviewer: ```suggestion
- Girar em 180° todas as páginas de um arquivo PDF:
```

`pdftk {{ficheiro.pdf}} cat {{1-endsouth}} output {{novo_ficheiro.pdf}}`

> - Rodar a terceira página de um ficheiro PDF 90 graus no sentido horário, não modificando as restantes:

Reviewer: ```suggestion
- Girar a terceira página de um arquivo PDF em 90° no sentido horário, não modificando as restantes:
```

# pdftk

> Conjunto de utilitários para manipular ficheiros PDF.
> Mas informação em: <https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit>.

> 

Reviewer 2: I don't think we're using 2 blank lines between the main description and the commands, but please correct me if I'm wrong :)

- Extrair conjuntos de páginas de um ficheiro PDF (páginas 1 a 3, 5 e 6 a 10) e guardá-las num novo ficheiro:

`pdftk {{ficheiro.pdf}} cat {{1-3 5 6-10}} output {{novo_ficheiro.pdf}}`

> - Juntar (concatenar) uma lista de ficheiros PDF guardando o resultado num novo ficheiro:

Reviewer 2: Uma sugestão mínima :)

```suggestion
- Juntar (concatenar) uma lista de ficheiros PDF, guardando o resultado num novo ficheiro:
```

`pdftk {{ficheiro1.pdf ficheiro2.pdf ficheiroN.pdf ...}} cat output {{novo_ficheiro.pdf}}`

> - Partir cada página de um ficheiro PDF num ficheiro separado, com um padrão para o nome do novo ficheiro:

Reviewer 2: ```suggestion
- Partir cada página de um ficheiro PDF num ficheiro separado, com um padrão para o nome dos novos ficheiros:
```

> `pdftk {{ficheiro.pdf}} burst output {{pagina_%d.pdf}}`

Reviewer 2: Estamos no século XXI, podemos usar carateres acentuados em filenames :)

```suggestion
`pdftk {{ficheiro.pdf}} burst output {{página_%d.pdf}}`
```

> - Rodar todas as páginas de um ficheiro PDF 180º:

Reviewer 2: Just a minor nitpick 🤓

```suggestion
- Rodar todas as páginas de um ficheiro PDF 180°:
```
